ngboost/distns/betabernoulli.py

```diff
@@ -0,0 +1,125 @@
+from scipy.special import polygamma, gamma, digamma
+from scipy.special import beta as betafunction
+from fastbetabino import *
+from array import array  
+import sys   
+class BetaBernoulliLogScore(LogScore): 
+    
+    def score(self, Y):
+        return -self.dist.logpmf(Y)
+
+    def d_score(self, Y):
+        D = np.zeros((len(Y), 2)) # first col is dS/d(log(α)), second col is dS/d(log(β))
+
+        D[:, 0] =   -self.alpha * (
+                        digamma(self.alpha + self.beta) + 
+                        digamma(Y + self.alpha) -
+                        digamma(self.alpha + self.beta + 1) -
+                        digamma(self.alpha)
+                    )
+        D[:, 1] =   -self.beta * (
+                        digamma(self.alpha + self.beta) + 
+                        digamma(-Y + self.beta + 1) -
+                        digamma(self.alpha + self.beta + 1) -
+                        digamma(self.beta)
+                    )
+        return D
+    
+    def metric(self):
+        FI = np.zeros((self.alpha.shape[0], 2, 2))
+        FI[:, 0, 0] =   ((self.alpha * (
+                        digamma(self.alpha + self.beta) + 
+                        digamma(0 + self.alpha) -
+                        digamma(self.alpha + self.beta + 1) -
+                        digamma(self.alpha)
+                        ))**2 * self.dist.pmf(0) + 
+                        (self.alpha * (
+                        digamma(self.alpha + self.beta) + 
+                        digamma(1 + self.alpha) -
+                        digamma(self.alpha + self.beta + 1) -
+                        digamma(self.alpha)
+                        ))**2 * self.dist.pmf(1))
+        FI[:, 1, 1] =   ((self.beta * (
+                        digamma(self.alpha + self.beta) + 
+                        digamma(-0 + self.beta + 1) -
+                        digamma(self.alpha + self.beta + 1) -
+                        digamma(self.beta)
+                        ))**2 * self.dist.pmf(0) + 
+                        (self.beta * (
+                        digamma(self.alpha + self.beta) + 
+                        digamma(-1 + self.beta + 1) -
+                        digamma(self.alpha + self.beta + 1) -
+                        digamma(self.beta)
+                        ))**2 * self.dist.pmf(1))
+        return FI
+class BetaBernoulli(RegressionDistn):
+
+    n_params = 2
+    scores = [BetaBernoulliLogScore]
+
+    def __init__(self, params):
+        # save the parameters
+        self._params = params
+        
+        # create other objects that will be useful later
+        self.log_alpha = params[0]
+        self.log_beta = params[1]
+        self.alpha = np.exp(self.log_alpha)
+        self.beta = np.exp(self.log_beta)
+        self.dist = dist(n=1, a=self.alpha, b=self.beta)
+
+    def sigmoid(self, x):
+        return 1/(1+np.exp(-x))
+
+    def fit(Y):
+        
+        def fit_alpha_beta_py(impressions, clicks, alpha0=1.5, beta0=5, niter=1000):
```

Comment:
Can we clean this function up a bit? In particular it's not clear what impressions / clicks are supposed to be. If `impressions` is going to be a vector of ones in all cases maybe we can remove it as an argument? 

Also it'd be great if we could apply `black` formatting.

Reply:
Cleared the function